<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>738</width>
    <height>451</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>738</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>&amp;Файл</string>
    </property>
    <addaction name="createDatabase"/>
    <addaction name="openDatabase"/>
    <addaction name="separator"/>
    <addaction name="createSensorsPosition"/>
    <addaction name="readSensorsPosition"/>
    <addaction name="saveSensorsPositionAs"/>
    <addaction name="separator"/>
    <addaction name="readGraphicsSet"/>
    <addaction name="saveGraphicsSetAs"/>
    <addaction name="separator"/>
    <addaction name="exit"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>&amp;Цвет</string>
    </property>
    <addaction name="setDefaultColor"/>
    <addaction name="setBlackWhiteColor"/>
    <addaction name="setManualColor"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>&amp;Задачи</string>
    </property>
    <addaction name="correctionRandA"/>
    <addaction name="separator"/>
    <addaction name="action_2"/>
    <addaction name="separator"/>
    <addaction name="location"/>
    <addaction name="filterOfEvent"/>
   </widget>
   <widget class="QMenu" name="menu_4">
    <property name="title">
     <string>&amp;Таблицы</string>
    </property>
    <addaction name="tableOfSignals"/>
    <addaction name="tableOfClasters"/>
    <addaction name="StatisticsByChannel"/>
    <addaction name="equipmentParameters"/>
    <addaction name="sensorsLocation"/>
    <addaction name="timeOfCommandStartPause"/>
    <addaction name="ASLbyChannel"/>
    <addaction name="find"/>
   </widget>
   <widget class="QMenu" name="menu_5">
    <property name="title">
     <string>&amp;Графики</string>
    </property>
    <addaction name="createGraphics"/>
    <addaction name="graphicsAnalysis"/>
    <addaction name="localizationIncrease"/>
    <addaction name="localizationReduce"/>
    <addaction name="zonalLocalization"/>
    <addaction name="separator"/>
    <addaction name="copy"/>
    <addaction name="saveInFile"/>
   </widget>
   <widget class="QMenu" name="menu_6">
    <property name="title">
     <string>&amp;Окно</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_7">
    <property name="title">
     <string>О &amp;программе</string>
    </property>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
   <addaction name="menu_4"/>
   <addaction name="menu_5"/>
   <addaction name="menu_6"/>
   <addaction name="menu_7"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="createDatabase"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="createDatabase">
   <property name="text">
    <string>Создать базу данных...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="openDatabase">
   <property name="text">
    <string>Открыть базу данных...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="createSensorsPosition">
   <property name="text">
    <string>Создать расстановку датчиков...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="readSensorsPosition">
   <property name="text">
    <string>Прочитать расстановку датчиков...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="saveSensorsPositionAs">
   <property name="text">
    <string>Сохранить расстановку датчиков как...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="readGraphicsSet">
   <property name="text">
    <string>Прочитать установки графиков...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+T</string>
   </property>
  </action>
  <action name="saveGraphicsSetAs">
   <property name="text">
    <string>Сохранить установки графиков как...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
  </action>
  <action name="setDefaultColor">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>По умолчанию</string>
   </property>
  </action>
  <action name="setBlackWhiteColor">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Чёрно-белый</string>
   </property>
  </action>
  <action name="setManualColor">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Установка...</string>
   </property>
  </action>
  <action name="exit">
   <property name="text">
    <string>Выход</string>
   </property>
  </action>
  <action name="correctionRandA">
   <property name="text">
    <string>Коррекция R и A</string>
   </property>
  </action>
  <action name="action_2">
   <property name="text">
    <string>Подготовка данных</string>
   </property>
  </action>
  <action name="location">
   <property name="text">
    <string>Локация...</string>
   </property>
  </action>
  <action name="filterOfEvent">
   <property name="text">
    <string>Фильтрация событий...</string>
   </property>
  </action>
  <action name="tableOfSignals">
   <property name="text">
    <string>Таблица сигналов</string>
   </property>
  </action>
  <action name="tableOfClasters">
   <property name="text">
    <string>Таблица кластеров</string>
   </property>
  </action>
  <action name="StatisticsByChannel">
   <property name="text">
    <string>Статистика по каналам</string>
   </property>
  </action>
  <action name="equipmentParameters">
   <property name="text">
    <string>Параметры аппаратуры</string>
   </property>
  </action>
  <action name="sensorsLocation">
   <property name="text">
    <string>Расположение приемников</string>
   </property>
  </action>
  <action name="timeOfCommandStartPause">
   <property name="text">
    <string>Время команд СТАРТ, ПАУЗА</string>
   </property>
  </action>
  <action name="ASLbyChannel">
   <property name="text">
    <string>ASL по каналам</string>
   </property>
  </action>
  <action name="find">
   <property name="text">
    <string>Найти...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="createGraphics">
   <property name="text">
    <string>Создание графиков...</string>
   </property>
  </action>
  <action name="graphicsAnalysis">
   <property name="text">
    <string>Графический анализ</string>
   </property>
  </action>
  <action name="localizationIncrease">
   <property name="text">
    <string>Локализация (увеличить)</string>
   </property>
  </action>
  <action name="localizationReduce">
   <property name="text">
    <string>Локализация (уменьшить)</string>
   </property>
  </action>
  <action name="zonalLocalization">
   <property name="text">
    <string>Зонная локация</string>
   </property>
  </action>
  <action name="copy">
   <property name="text">
    <string>Копировать</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+K</string>
   </property>
  </action>
  <action name="saveInFile">
   <property name="text">
    <string>Сохранить в файл...</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
